tox.ini

```diff
@@ -57,6 +57,8 @@ deps =
     astropy40: astropy==4.0.*
     astropylts: astropy==5.0.*
 
+    oldestdeps: numpy==1.19.* astropy==3.*
```

Comment:
I think you need to define each dependency on two separate oldestdeps: lines

Reply:
Fixed.